docs/Data_Submission_Portal/Users_Guide/Troubleshooting.md

```diff
@@ -0,0 +1,1236 @@
+| __No link destination found for {}__ | The parent entity's submitter_id does not exist | Make sure the child entity is linking to the correct parent entity and that the submitter_id is accurate |
+
+```Request1
+{
+  "read_groups": {
+    "submitter_id": "Read_group_00093"
+  }, 
+  "data_type": "Aligned Reads", 
+  "md5sum": "aa6e82d11ccd8452f813a15a6d84faf1", 
+  "type": "submitted_aligned_reads", 
+  "data_category": "Sequencing Reads", 
+  "data_format": "BAM",
+  "project_id": "GDC-INTERNAL", 
+  "file_size": 928374, 
+  "file_name": "test.bam", 
+  "experimental_strategy": "WGS", 
+  "submitter_id": "Blood-00009-aliquot01_lane1_barcodeACGTAC_55.bam"
+}
+```
+```Response1
+{
+  "cases_related_to_created_entities_count": 0,
+  "cases_related_to_updated_entities_count": 1,
+  "code": 400,
+  "created_entity_count": 0,
+  "entities":
+    [
+        {
+            "action":"update",
+            "errors":
+                [
+                    {
+                        "keys":["read_groups"],
+                        "message":"'read_groups' link has to be one_to_many, target node read_group already has submitted_aligned_reads_files",
+                        "type":"INVALID_LINK"
+                    }
+                ],
+            "id":
+                [
+                    {
+                        "id":"a00f076e-d694-47dd-8e50-24c28e90fd6a","submitter_id":"GDC-INTERNAL-000093"
+                    },
+                    {
+                        "id":"8c7e2ce7-a772-441c-a33b-e0e3bdbd4f78","submitter_id":"GDC-INTERNAL-000078"
+                    }
+                ],
+            "related_cases":[],
+            "type":"submitted_aligned_reads",
+            "unique_keys":
+                [
+                    {
+                        "project_id":"GDC-INTERNAL",
+                        "submitter_id":"Blood-00009-aliquot01_lane1_barcodeACGTAC_55.bam"
+                    }
+                ],
+            "valid":false,
+            "warnings":[]
+        }
+    ],
+    "entity_error_count":1,
+    "message":"Transaction aborted due to 1 invalid entity.",
+    "success":false,
+    "transaction_id":6372988,
+    "transactional_error_count":0,
+    "transactional_errors":[],
+    "updated_entity_count":0
+}
+```
+
+```Request1
+[
+    {
+        "data_category": "Sequencing Reads",
+        "data_format": "BAI",
+        "data_type": "Aligned Reads Index",
+        "gencode_version": "v36",
+        "project_id": "CPTAC-3",
+        "submitter_id": "Aligned_Reads_Index_000000",
+        "file_name": "1c5321a4-3b33-4787-9393-3c999940284f.rna_seq.genomic.gdc_realn.bam.bai",
+        "file_size": 5106896,
+        "md5sum": "250a27e69c3556311934d0d8daab2207",
+        "urls": "s3://cleversafe.service.consul/stage-submission-5/1c5321a4-3b33-4787-9393-3c999940284f/1c5321a4-3b33-4787-9393-3c999940284f.rna_seq.genomic.gdc_realn.bam.bai",
```

Comment:
Let's remove any instance of the "urls" and "gencode_version"  fields.  End users will never submit these fields and we don't want them to know our  cleversafe URL information.